src/LargeXlsx/Util.cs

```diff
@@ -28,107 +28,112 @@ OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN
-                else if (c == '\'') sb.Append("&apos;");
-                else if (c == '"') sb.Append("&quot;");
-                else sb.Append(c);
-            }
-
-            return sb.ToString();
-        }
-
-        public static string GetColumnName(int columnIndex)
-        {
-            if (columnIndex < 1 || columnIndex > MaxColumnCount)
-                throw new InvalidOperationException($"A worksheet can contain at most {MaxColumnCount} columns ({columnIndex} attempted)");
-            var columnName = CachedColumnNames[columnIndex - 1];
-            if (columnName == null)
-            {
-                columnName = GetColumnNameInternal(columnIndex);
-                CachedColumnNames[columnIndex - 1] = columnName;
-            }
-            return columnName;
-        }
-
-        private static string GetColumnNameInternal(int columnIndex)
-        {
-            var columnName = new StringBuilder(3); // This has been measured to be faster than string concatenation
-            while (true)
-            {
-                if (columnIndex > 26)
-                {
-                    columnIndex = Math.DivRem(columnIndex - 1, 26, out var rem);
-                    columnName.Insert(0, (char)('A' + rem));
-                }
-                else
-                {
-                    columnName.Insert(0, (char)('A' + columnIndex - 1));
-                    return columnName.ToString();
-                }
-            }
-        }
-
-        public static double DateToDouble(DateTime date)
-        {
-            var days = date.Subtract(ExcelEpoch).TotalDays + 1;
-            // Excel wrongly assumes that 1900 is a leap year:
-            // https://docs.microsoft.com/en-us/office/troubleshoot/excel/wrongly-assumes-1900-is-leap-year
-            if (date >= Date19000301) days++;
-            return days;
-        }
-
-        public static int BoolToInt(bool value) => value ? 1 : 0;
-
-        public static string EnumToAttributeValue<T>(T enumValue)
-        {
-            var s = enumValue.ToString();
-            return char.ToLowerInvariant(s[0]) + s.Substring(1);
-        }
-
-        // Hashing procedure courtesy of https://docs.microsoft.com/en-us/openspecs/office_file_formats/ms-offcrypto/1357ea58-646e-4483-92ef-95d718079d6f
-        public static byte[] ComputePasswordHash(string password, byte[] saltValue, int spinCount)
-        {
-            var hasher = new SHA512Managed();
-            var hash = hasher.ComputeHash(saltValue.Concat(Encoding.Unicode.GetBytes(password)).ToArray());
-            for (var i = 0; i < spinCount; i++)
-            {
-                var iterator = BitConverter.GetBytes(i);
-                if (!BitConverter.IsLittleEndian)
-                    Array.Reverse(iterator);
-                hash = hasher.ComputeHash(hash.Concat(iterator).ToArray());
-            }
-            return hash;
-        }
-    }
+	internal static class Util
```

Comment:
Unfortunately, the change in indentation width is not helpful to identify your changes here (meld helped :smile:), but above all it is not consistent with the rest of the code.